filebeat/input/log/input.go

```diff
@@ -718,6 +718,8 @@ func (p *Input) Wait() {
 
 // Stop stops all harvesters and then stops the input
 func (p *Input) Stop() {
+	defer logp.Debug("input", "Log input stopped")
+
```

Comment:
make sense to keep a trace of that when debugging why events are not going through..